erpnext/crm/doctype/appointment_booking_settings/appointment_booking_settings.js

```diff
@@ -0,0 +1,13 @@
+// frappe.ui.form.on('Availability Of Slots', 'from_time', check_time)
+// frappe.ui.form.on('Availability Of Slots', 'to_time', check_time)
+
+frappe.ui.form.on('Appointment Booking Settings', 'validate',check_times);
+function check_times(frm) {
+	$.each(frm.doc.availability_of_slots || [], function (i, d) {
+		let from_time = Date.parse('01/01/2019 ' + d.from_time);
+		let to_time = Date.parse('01/01/2019 ' + d.to_time);
+		if (from_time > to_time) {
+			frappe.throw(__(`In row ${i + 1} of Availability Of Slots : "To Time" must be later than "From Time"`));
```

Comment:
```suggestion
			frappe.throw(__(`In row {0} of Availability Of Slots : "To Time" must be later than "From Time"`, [i+1]));
```

ref: https://frappe.io/docs/user/en/translations#2-variables